Makefile

```diff
@@ -141,3 +157,19 @@ $(STANC):
 	curl -L https://github.com/stan-dev/stanc3/releases/download/$(STANC3_VERSION)/$(OS_TAG)$(ARCH_TAG)-stanc -o $(STANC) --retry $(STANC_DL_RETRY) --retry-delay $(STANC_DL_DELAY)
 	chmod +x $(STANC)
 endif
+
+##
+# This is only run if the `include` statements earlier fail to find a file.
+# We assume that means the submodule is missing
+##
+$(MATH)make/% :
+	@echo 'ERROR: Missing Stan submodules.'
+	@echo 'We tried to find the Stan Math submodule at:'
+	@echo '  $(MATH)'
+	@echo ''
+	@echo 'The most likely source of the problem is BridgeStan was cloned without'
+	@echo 'the --recursive flag.  To fix this, run the following command:'
+	@echo '  git submodule update --init --recursive'
```

Comment:
Is there anway, maybe via extra bits in our top level `.gitmodules` file, to include `--depth=1`, `--recurse-submodules`, and `--shallow-submodules`?  I read the man page on `git-submodule`, but didn't come away with an obvious answer.

Reply:
You can put `shallow = true` in there but it sounds like that [may be buggy](https://stackoverflow.com/a/47374702)

Reply:
Ok.  Thanks for double checking.